addons/hr_work_entry_holidays/models/hr_leave.py

```diff
@@ -169,7 +169,10 @@ def write(self, vals):
 
         start = min(self.mapped('date_from') + [fields.Datetime.from_string(vals.get('date_from', False)) or datetime.max])
         stop = max(self.mapped('date_to') + [fields.Datetime.from_string(vals.get('date_to', False)) or datetime.min])
-        with self.env['hr.work.entry']._error_checking(start=start, stop=stop, skip=skip_check):
+        employee_ids = self.employee_id.ids
+        if 'employee_id' in vals and vals['employee_id']:
+            employee_ids += vals['employee_id']
```

Comment:
Since `employee_id` is an integer and the operator `+=` expects a list, this will fail:


`TypeError: 'int' object is not iterable`

The same occurs in the other file.

CC @rodrigosu-vauxoo @desdelinux

Reply:
Indeed, thanks for the report!
Will be fixed in https://github.com/odoo/odoo/pull/117097